x-pack/platform/plugins/shared/saved_objects_tagging/public/components/base/tag_selector.tsx

```diff
@@ -58,6 +58,7 @@ const renderCreateOption = () => {
       gutterSize="xs"
       alignItems="center"
       responsive={false}
+      title=""
```

Comment:
This one is definitely a strange one, however I wonder if it would make more sense to simply provide the `create tag` title so we might have something like so; 
<img width="512" height="82" alt="Screenshot 2025-08-27 at 10 14 11" src="https://github.com/user-attachments/assets/8d62e539-608b-4977-8b6c-c4a8e491dae4" />

as opposed to an empty value, that might could be easily removed, in turn causing the same issue again